installation/scripts/install-kyma.sh

```diff
@@ -45,6 +45,8 @@ ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
 INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/kyma/installer-cr-kyma-dependencies.yaml
 OVERRIDES_COMPASS_GATEWAY="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-compass-gateway.yaml
 API_GATEWAY_OVERRIDES="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-api-gateway.yaml
+TRACING_OVERRIDES="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-tracing.yaml
```

Comment:
Not needed if other comments are addressed.

Reply:
Not applicable